### ISSUE TITLE
Clicking annotate no longer opens sidebar on PDFs (in old PDF.js versions)

### ISSUE THREAD

**Issue by dwhly** [hypothesis/product-backlog#1205]
### Steps to reproduce
1. Select text, for instance on the embedded PDF here https://paleorxiv.org/3auze  (Also, same issue happens on a PDF dragged to docdrop.org)
2. Click annotate

### Expected behaviour
Sidebar should open

### Actual behaviour
It doesn't.
Also, note that clicking on the text to close the sidebar doesn't work either-- related?

### Browser/system information
Chrome

### Additional details
Version: 1.774.0
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36
URL: https://paleorxiv.org/3auze
Fingerprint: 76325b07dbc7d341a3e486d8951f711cfb6332ef96f44773fdeca5f24400b7c0
Account: Dan Whaley (acct:dwhly@hypothes.is)
Date: Fri May 07 2021 07:46:11 GMT-0700 (Pacific Daylight Time)

**Comment by klemay** [comment]
-  I cannot reproduce if I open a PDF in my browser and activate the extension
- Dan can repro in docdrop
- We cannot repro in LMS

**Comment by robertknight** [comment]
I can reproduce the issue on https://paleorxiv.org/3auze. There is an error in the terminal:

<img width="897" alt="pdfContainer error" src="https://user-images.githubusercontent.com/2458/117468863-07045880-af4d-11eb-9d40-23625794c44a.png">

**Comment by robertknight** [comment]
The `this.pdfContainer` element is initialized with:

```js
    // Get the element that contains all of the PDF.js UI. This is typically
    // `document.body`.
    this.pdfContainer = window_.PDFViewerApplication?.appConfig?.appContainer;
```

On this page `PDFViewerApplication` exists, but the `appConfig` property does not. The issue might relate to the version of PDF.js in use.

**Comment by dwhly** [comment]
I believe docdrop (https://github.com/dwhly-proj/droppdf) has just been updated to recent PDFjs? cc @genuinebuildmonkey?

**Comment by robertknight** [comment]
In Via's version of PDF.js:

<img width="755" alt="pdfContainer Via" src="https://user-images.githubusercontent.com/2458/117469467-ac1f3100-af4d-11eb-8582-4e0ea004ab01.png">

The version of PDF.js can be checked by evaluating `window.PDFJS.version` in the console. PaleoarXiv is using a much older version, 1.0.1145. This might be an old version of https://github.com/hypothesis/pdf.js-hypothes.is.

**Comment by dwhly** [comment]
I will let the OSF team know to check the version of PDFjs.  Also, will doublecheck docdrop.  ty!

**Comment by robertknight** [comment]
From what I've been able to determine from the PDF.js project history, this issue affects PDF.js versions before v1.5.188 that was released on 2016-04-21. All of the PDF.js viewers that we ship are newer (various 2.x versions) but there may still be some publisher sites using older releases of https://github.com/hypothesis/pdf.js-hypothes.is.

There are some general related issues about figuring out our support policy for older PDF.js releases and approach to deprecating any releases that we decide to no longer support: https://github.com/hypothesis/client/issues/2644, https://github.com/hypothesis/client/issues/2643.
